cmake/BTApplication.cmake

```diff
@@ -9,6 +9,7 @@ FIND_PACKAGE(Qt5Gui ${REQUIRED_QT_VERSION} REQUIRED)
 FIND_PACKAGE(Qt5Widgets ${REQUIRED_QT_VERSION} REQUIRED)
 FIND_PACKAGE(Qt5Xml ${REQUIRED_QT_VERSION} REQUIRED)
 FIND_PACKAGE(Qt5Test ${REQUIRED_QT_VERSION} REQUIRED)
+FIND_PACKAGE(Qt5TextToSpeech ${REQUIRED_QT_VERSION} REQUIRED)
```

Comment:
Can this be made configurable using a compile-time flag to CMake?

Reply:
Done, there's now `-DBUILD_TEXT_TO_SPEECH=OFF` to disable the feature.